packages/cpp/src/browser/cpp-language-client-contribution.ts

```diff
@@ -67,12 +67,17 @@ export class CppLanguageClientContribution extends BaseLanguageClientContributio
     }
 
     protected get documentSelector() {
+        // This is used (at least) to determine which files, when they are open,
+        // trigger the launch of the C/C++ language server.
         return HEADER_AND_SOURCE_FILE_EXTENSIONS;
     }
 
     protected get globPatterns() {
+        // This is used (at least) to determine which files we watch.  Change
+        // notifications are forwarded to the language server.
         return [
-            '**/*.{' + HEADER_AND_SOURCE_FILE_EXTENSIONS.join() + '}'
+            '**/*.{' + HEADER_AND_SOURCE_FILE_EXTENSIONS.join() + '}',
```

Comment:
Would it be better to have it explicitly like `.join(',')` ?
I had to test it in my interpreter because I thought the default was `.join('')`...

Reply:
Yes, I agree it would be clearer.  It's out of the scope of this patch though.